### PR TITLE
initial commit for playerstats module

### PR DESCRIPTION
**Pull request by joooooooooe-star**
my first try at playerstats module though i forgot where i got the reference

### CODE REVIEW

File: nari/io/reader/actlogutils/playerstats.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    param_ints = (int(param) for param in params if param != "0")
+    player_stats = dict(zip(stats_definitions, param_ints))
+
+    # Malformed line check
+    if len(player_stats) != 17:
+        raise ActLineReadError("Params are unexpectedly short")
+
+    # clear the blank field
+    try:
+        del player_stats["delete"]
```
Comment (lunarrize) [inline nari/io/reader/actlogutils/playerstats.py:44]: You need to remove this since this will always raise an exception now.

**Review by lunarrize — CHANGES_REQUESTED** [review]
small change but otherwise good

File: nari/io/reader/actlogutils/playerstats.py (hunk 1 above)
Comment (joooooooooe-star) [reply nari/io/reader/actlogutils/playerstats.py]: this is old and many other things were fixed

**Review by lunarrize — COMMENTED** [review]
👍

**Pull request merged**